docs/nuget-org/individual-accounts.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Individual accounts
+description: Individual acccounts on NuGet.org are required to publish packages
+author: mikejo5000
+ms.author: mikejo
+ms.date: 06/05/2019
+ms.topic: conceptual
+---
+
+# Individual accounts
+
+You must create an individual account to publish and manage packages on NuGet.org.
+
+## Individual accounts vs. organization accounts
+
+Your individual (user) account is your identity on NuGet.org and can be a member of any number of organizations. A package can belong to an organization account like it can belong to an individual account. Package consumers don't see any difference between an individual account or the organization account: both appear as package `owners`.
+
+An organization account has one or more individual accounts as its members. These members can manage a set of packages while maintaining a single identity for ownership.
+
+## Add a new individual account
+
+To create a NuGet.org account, you need to have a personal Microsoft account (MSA) or an Azure Active Directory (AAD) account. If you do not have one, you can [create](https://signup.live.com) one. Follow the following steps if you have an MSA or AAD account.
+
+1. Go to the [NuGet.org login page](https://www.nuget.org/users/account/LogOn).
+
+1. Click on **Sign in with Microsoft** button.
+
+1. Enter your Microsoft account or Azure ActiveDirectory account details.
+
+1. Please click **Yes** to accept the permissions to be given to the *NuGet.org* application.
+
+   ![Giving permissions to NuGet.org](media/nuget-org-permissions.png)
```

Comment:
This image seems too big for the page. Can we replace this with a screenshot such that the permissions view looks closer to the real size and not enlraged?